coderd/workspacebuilds.go

```diff
@@ -51,6 +51,7 @@ func (api *API) workspaceBuild(rw http.ResponseWriter, r *http.Request) {
 		data.metadata,
 		data.agents,
 		data.apps,
+		data.templateVersions[0],
```

Comment:
This will panic if the length is not at least 1. That may be fine if we have the expectation that this is always true, but wanted to point it out.

Reply:
Yes, we always expect to have at least 1. Thanks for pointing it out.